frontend/components/text-classifier/results/RecordTextClassification.vue

```diff
@@ -34,7 +34,7 @@
         @annotate="onAnnotate"
         @edit="onEdit"
       />
-      <ClassifierExplorationArea v-else :labels="predictionLabels" />
+      <ClassifierExplorationArea v-else :labels="predictionLabels" :predicted-as="record.predicted_as" />
     </div>
     <div v-if="!annotationEnabled" class="record__labels">
```

Comment:
I'm a bit confused here. Regarding the PR changes, the `ClassifierExplorationArea` component handles already a LabelPill for label rendering. 

What this extra `LabelPill` section does? Is shown in explore views for records with annotation but making some tests, I cannot render the component in view.

Reply:
@frascuchon There are two label pills that were visually the same before, those of the predictions, in white with a grey border and the small grey ones that indicate the annotation on the right side of the record.

Reply:
Maybe the small pills in grey should be another component, `tag` for example